components/EventFeed.vue

```diff
@@ -0,0 +1,176 @@
+    default:
+      return "";
+  }
+}
+
+function getNotifAvatar(type: string): string {
+  switch (type) {
+    case "scanned":
+      return "/icons/notifications/scanned.png";
+    case "cookfail":
+      return "/icons/notifications/cookfail.png";
+    case "cook":
+      return "/icons/notifications/cook.png";
+    case "release":
+      return "/icons/notifications/release.png";
+    case "move":
+      return "/icons/notifications/move.png";
+    case "claim":
+      return "/icons/notifications/claim.png";
+    case "drop":
+      return "/icons/notifications/drop.png";
+    case "gift":
+      return "/icons/notifications/gift.png";
+    case "spawn":
+      return "/icons/notifications/spawn.png";
+    default:
+      return "";
+  }
+}
+
+function getNotifColors(type: string): { background: string; text: string } {
+  switch (type) {
+    case "scanned":
+      return { background: "bg-notif-scanned dark:bg-notif-scanned", text: "text-notif-black dark:text-notif-black" };
+    case "cookfail":
+      return { background: "bg-notif-cookfail dark:bg-notif-cookfail", text: "text-notif-white dark:text-notif-white" };
+    case "cook":
+      return { background: "bg-notif-cook dark:bg-notif-cook", text: "text-notif-black dark:text-notif-black" };
+    case "release":
+      return { background: "bg-notif-release dark:bg-notif-release", text: "text-notif-white dark:text-notif-white" };
+    case "move":
+      return { background: "bg-notif-move dark:bg-notif-move", text: "text-notif-white dark:text-notif-white" };
+    case "claim":
+      return { background: "bg-notif-claim dark:bg-notif-claim", text: "text-notif-black dark:text-notif-black" };
+    case "drop":
+      return { background: "bg-notif-drop dark:bg-notif-drop", text: "text-notif-white dark:text-notif-white" };
+    case "gift":
+      return { background: "bg-notif-gift dark:bg-notif-gift", text: "text-notif-white dark:text-notif-white" };
+    case "spawn":
+      return { background: "bg-notif-spawn dark:bg-notif-spawn", text: "text-notif-white dark:text-notif-white" };
+    default:
+      return { background: "", text: "" };
+  }
+}
+
+function getNotifProgressBg(type: string): string {
+  switch (type) {
+    case "scanned":
+      return "bg-notif-black dark:bg-notif-black";
+    case "cookfail":
+      return "bg-notif-white dark:bg-notif-white";
+    case "cook":
+      return "bg-notif-black dark:bg-notif-black";
+    case "release":
+      return "bg-notif-white dark:bg-notif-white";
+    case "move":
+      return "bg-notif-white dark:bg-notif-white";
+    case "claim":
+      return "bg-notif-black dark:bg-notif-black";
+    case "drop":
+      return "bg-notif-white dark:bg-notif-white";
+    case "gift":
+      return "bg-notif-white dark:bg-notif-white";
+    case "spawn":
+      return "bg-notif-white dark:bg-notif-white";
+    default:
+      return "";
+  }
+}
```

Comment:
minor, but could this be made a bit more simpler to reduce the repetition.

Maybe set 1 type as the `default` and then group the remaining and return the other value?

Reply:
Addressed here, thanks! https://github.com/whitep4nth3r/pantherworld/commit/1d52147ba7dc934498763c547af39de844045355